VISION.md

```diff
@@ -0,0 +1,41 @@
+### The Danger Vision
+
+Danger is a long term project, from our current viewpoint her role is
+
+ * Provide the best API for communication in Continuous Integration.
```

Comment:
I actually would describe Danger as 'an extra reviewer for projects', allowing it to check various things specified by the Dangerfile DSL.